kf_model_fhir/app.py

```diff
@@ -333,10 +343,9 @@ def _update_ig_config(
             resources_dict[entry['reference']['reference']] = entry
         else:
             del rd[entry['reference']['reference']]
-
-        if rm_file:
-            os.rmfile(rd['filepath'])
-            logger.info(f'🗑 Deleted resource file {rd["filepath"]}')
+            if rm_file:
```

Comment:
FYI - This change is not related to the issues addressed in this PR. Just noticed that this blob should have been indented. Didn't want to create a whole separate PR for it.